### PR TITLE
Adding teradata to landing pages

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Adding Teradata cloud adapter to adapter landing pages

## Checklist
- [ ] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [ ] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

File: website/docs/docs/cloud/connect-data-platform/about-connections.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ dbt Cloud can connect with a variety of data platform providers including:
 - [PostgreSQL](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb)
 - [Snowflake](/docs/cloud/connect-data-platform/connect-snowflake)
 - [Starburst or Trino](/docs/cloud/connect-data-platform/connect-starburst-trino)
-
+- [Teradata](/docs/cloud/connect-data-platform/connect-teradata) <Lifecycle status="Preview" />
```
Comment (nghi-ly) [inline website/docs/docs/cloud/connect-data-platform/about-connections.md:21]: minor Markdown sugg so there's a line break btn list and sentence

```suggestion
- [Teradata](/docs/cloud/connect-data-platform/connect-teradata) <Lifecycle status="Preview" />

```

File: website/snippets/_adapters-trusted.md [hunk 2]
```diff
@@ -94,8 +94,8 @@
 
 <Card
     title="Teradata"
-    body="<ul><li><a href='/docs/core/connect-data-platform/teradata-setup'>Install with dbt Core  </a> </li> </ul><br /><br /> <a href=https://badge.fury.io/py/dbt-teradata><img src=https://badge.fury.io/py/dbt-teradata.svg/></a>"
-    pills='["dbt Core"]'
+    body="<ul><li><a href='/docs/cloud/connect-data-platform/connect-teradata'>Set up in dbt Cloud  </a> <br /></li> <li><a href='/docs/core/connect-data-platform/teradata-setup'>Install with dbt Core  </a> </li> </ul><br /><br /> <a href=https://badge.fury.io/py/dbt-teradata><img src=https://badge.fury.io/py/dbt-teradata.svg/></a>"
+    pills='["dbt Cloud", "dbt Core"]'
```
Comment (nghi-ly) [inline website/snippets/_adapters-trusted.md:98]: sugg fixing the PY and pills alignment. it's shifted down compared to the other cards in the same row:

 
Screenshot 2024-10-07 at 12 06 30 PM

**Review by nghi-ly — COMMENTED** [review]
love seeing another card here! thanks for working on this! left a cpl suggs

File: website/snippets/_adapters-trusted.md [hunk 3]
```diff
@@ -94,8 +94,8 @@
 
 <Card
     title="Teradata"
-    body="<ul><li><a href='/docs/core/connect-data-platform/teradata-setup'>Install with dbt Core  </a> </li> </ul><br /><br /> <a href=https://badge.fury.io/py/dbt-teradata><img src=https://badge.fury.io/py/dbt-teradata.svg/></a>"
-    pills='["dbt Core"]'
+    body="<ul><li><a href='/docs/cloud/connect-data-platform/connect-teradata'>Set up in dbt Cloud  </a> <br /></li> <li><a href='/docs/core/connect-data-platform/teradata-setup'>Install with dbt Core  </a> </li> </ul><br /><br /> <a href=https://badge.fury.io/py/dbt-teradata><img src=https://badge.fury.io/py/dbt-teradata.svg/></a>"
```
Comment (matthewshaver) [inline website/snippets/_adapters-trusted.md]:
```suggestion
    body="<ul><li><a href='/docs/cloud/connect-data-platform/connect-teradata'>Set up in dbt Cloud  </a> <br /></li> <li><a href='/docs/core/connect-data-platform/teradata-setup'>Install with dbt Core  </a> </li> </ul><br /> <a href=https://badge.fury.io/py/dbt-teradata><img src=https://badge.fury.io/py/dbt-teradata.svg/></a>"
```

File: website/snippets/_adapters-trusted.md [hunk 4]
```diff
@@ -94,8 +94,8 @@
 
 <Card
     title="Teradata"
-    body="<ul><li><a href='/docs/core/connect-data-platform/teradata-setup'>Install with dbt Core  </a> </li> </ul><br /><br /> <a href=https://badge.fury.io/py/dbt-teradata><img src=https://badge.fury.io/py/dbt-teradata.svg/></a>"
-    pills='["dbt Core"]'
+    body="<ul><li><a href='/docs/cloud/connect-data-platform/connect-teradata'>Set up in dbt Cloud  </a> <br /></li> <li><a href='/docs/core/connect-data-platform/teradata-setup'>Install with dbt Core  </a> </li> </ul><br /> <a href=https://badge.fury.io/py/dbt-teradata><img src=https://badge.fury.io/py/dbt-teradata.svg/></a>"
```
Comment (matthewshaver) [inline website/snippets/_adapters-trusted.md]:
```suggestion
    body="<ul><li><a href='/docs/cloud/connect-data-platform/connect-teradata'>Set up in dbt Cloud  </a> <br /></li> <li><a href='/docs/core/connect-data-platform/teradata-setup'>Install with dbt Core  </a> </li> </ul><a href=https://badge.fury.io/py/dbt-teradata><img src=https://badge.fury.io/py/dbt-teradata.svg/></a>"
```

File: website/snippets/_adapters-trusted.md [hunk 5]
```diff
@@ -94,8 +94,8 @@
 
 <Card
     title="Teradata"
-    body="<ul><li><a href='/docs/core/connect-data-platform/teradata-setup'>Install with dbt Core  </a> </li> </ul><br /><br /> <a href=https://badge.fury.io/py/dbt-teradata><img src=https://badge.fury.io/py/dbt-teradata.svg/></a>"
-    pills='["dbt Core"]'
+    body="<ul><li><a href='/docs/cloud/connect-data-platform/connect-teradata'>Set up in dbt Cloud  </a> <br /></li> <li><a href='/docs/core/connect-data-platform/teradata-setup'>Install with dbt Core  </a> </li> </ul><a href=https://badge.fury.io/py/dbt-teradata><img src=https://badge.fury.io/py/dbt-teradata.svg/></a>"
```
Comment (nghi-ly) [inline website/snippets/_adapters-trusted.md]: overcorrected, maybe?

Screenshot 2024-10-07 at 12 36 02 PM

**Review by nghi-ly — COMMENTED** [review]
just one more!

**Review by nghi-ly — APPROVED** [review]
🚢

**Pull request merged**